src/resources/genomicsdb_import_config.proto

```diff
@@ -44,15 +44,6 @@ message Partition {
   optional int64 end = 5 [default = 0x7FFFFFFFFFFFFFFF ];
 }
 
-message GATK4Integration {
-  optional string output_vidmap_json_file = 1;
-  optional string output_callsetmap_json_file = 2;
-  optional int32 batch_size = 3 [default = 0];
-  optional bool use_samples_in_order_provided = 4 [default = false];
-  optional int64 lower_sample_index = 5 [default = 0];
-  optional int64 upper_sample_index = 6 [default = 0x7FFFFFFFFFFFFFFF];
-}
-
 message ImportConfiguration {
```

Comment:
Is there a way to define multiple types of import configuration? I'm seeing plenty of optional attributes for the class, is there a way to group them based on their lvl of cohesion and make classes which encapsulate them optional?